N36: Simon game (#1122)
rust-gamedev/rust-gamedev.github.io: content/news/036/index.md

[Tetra]: https://github.com/17cupsofcoffee/tetra
[steamworks]: https://crates.io/crates/steamworks

### [Simon Arcade]

![Simon arcade gameplay with arrows and buttons in different colors](simon.gif)
_a Simon Arcade gameplay with cheats enabled_

Based on [Simon (Original)], made with [Bevy],
the goal of this game is to push buttons in the correct order,
in an ever-increasing sequence.

This game was made to fit with the [Rust Arcade Cabinet]

> and will be available at [RustConf Portland] on August 5th 2022.

Author: ```suggestion
and was showcased at [RustConf Portland] on August 5th 2022.
```
